pkgs/applications/misc/elektroid/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gtk3
+, wrapGAppsHook
+, autoreconfHook
+, pkg-config
+, gettext
+, json-glib
+, automake
+, autoconf
+, libtool
+, alsa-lib
+, libpulseaudio
+, libsndfile
+, libsamplerate
+, zlib
+, libjson
+, libzip
+}:
+
+stdenv.mkDerivation rec {
+  pname = "elektroid";
+  version = "2.1";
+
+  src = fetchFromGitHub {
+    owner = "dagargo";
+    repo = pname;
+    rev = "${version}";
+    sha256 = "sha256-T7jxtmxlFe5WPC2KYDN6jDHwuVkmA6Y/h9qizZFH5rk=";
+  };
+
+  buildInputs = [
+    wrapGAppsHook
+    autoreconfHook
+    gtk3
+    pkg-config
+    gettext
+    json-glib
+    automake
+    autoconf
+    libtool
+    alsa-lib
+    libpulseaudio
+    libsndfile
+    libsamplerate
+    zlib
+    libjson
+    libzip
+  ];
+
+  meta = with lib; {
+    description = "Transfer application for Elektron devices";
+    homepage = "https://github.com/dagargo/elektroid";
+    maintainers = with maintainers; [ dag-h ];
+    license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
Thanks for the review! I fixed up the requested changes. What do I do with the new commits? Should they be squashed into one "elektroid: init at 2.1" commit and force-pushed to my fork?